examples/test/scatter.html

```diff
@@ -0,0 +1,82 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <title>MathBox - Scatter</title>
+    <script
+      type="text/javascript"
+      src="https://cdn.jsdelivr.net/npm/three@0.137.0/build/three.min.js"
+    ></script>
+    <script
+      type="text/javascript"
+      src="https://cdn.jsdelivr.net/npm/three@0.137.0/examples/js/controls/OrbitControls.js"
+    ></script>
+    <!--
+       - a minified version mathbox.min.js is also available;
+       - recommend using a specific version (not @latest) in public sites
+    -->
+    <script
+      type="text/javascript"
+      src="https://cdn.jsdelivr.net/npm/mathbox@latest/build/bundle/mathbox.js"
+    ></script>
+
+    <link rel="stylesheet" href="../../build/mathbox.css" />
+    <meta name="viewport" content="initial-scale=1, maximum-scale=1" />
+  </head>
+  <body>
+    <script>
+      var mathbox = MathBox.mathBox({
+        plugins: ["core", "controls", "cursor"],
+        controls: {
+          klass: THREE.OrbitControls,
+        },
+      });
+      var three = mathbox.three;
+
+      three.camera.position.set(2.5, 1, -1.3);
+      three.renderer.setClearColor(new THREE.Color(0x000000), 1.0);
+
+      var view = mathbox.cartesian({
+        range: [
+          [-1, 1],
+          [-1, 1],
+          [-1, 1],
+        ],
+        scale: [1, 1, 1],
+      });
+
+      view.grid({
+        width: 5,
+        opacity: 0.5,
+        axes: [1, 3],
+        blending: "add",
+      });
+
+      let positions = [],
+        colors = [],
+        n = 10000;
+      for (var i = 0; i <= n; ++i) {
+        let x = Math.random() * 2 - 1;
+        let y = (Math.random() * 2 - 1 + x) / 2;
+        let z = (Math.random() * 2 - 1 + y) / 2;
+
+        positions.push([x, y, z]);
+        colors.push([x, y, z, 1]);
+      }
+
+      view.array({
+        data: positions,
+        channels: 3,
+      });
+      view.array({
+        data: colors,
+        channels: 4,
+      });
+      view.point({
+        size: 10,
+        points: "<<",
+        colors: "<",
+      });
```

Comment:
```suggestion
        color: 0xFFFFFF
      });
```

Reply:
For the nice bright colors...

Maybe this should happen by default if `colors` is provided and `color` is not...

Reply:
I agree!